tester_app/client/app/bundles/HelloWorld/startup/helloWorldAppClient.jsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { Provider } from 'react-redux';
+
+import createStore from '../store/helloWorldStore';
+import HelloWorld from '../containers/HelloWorld';
+
+const helloWorldAppClient = props => {
```

Comment:
Hey -- after lasts nights talk with @samnang and @alexfedoseev  , we want to capitalize, @robwise, both the files and the class names.

Capitals are OK for an import that is either a class or generator function. Basically, both classes and generator functions make new instances.